data/src/main/java/com/teamagam/gimelgimel/data/response/entity/visitor/IResponseVisitor.java

```diff
@@ -0,0 +1,26 @@
+package com.teamagam.gimelgimel.data.response.entity.visitor;
+
+import com.teamagam.gimelgimel.data.response.entity.AlertResponse;
+import com.teamagam.gimelgimel.data.response.entity.GeometryResponse;
+import com.teamagam.gimelgimel.data.response.entity.ImageResponse;
+import com.teamagam.gimelgimel.data.response.entity.TextResponse;
+import com.teamagam.gimelgimel.data.response.entity.UserLocationResponse;
+import com.teamagam.gimelgimel.data.response.entity.VectorLayerResponse;
+import com.teamagam.gimelgimel.data.response.entity.UnknownResponse;
+
+public interface IResponseVisitor {
```

Comment:
If you want, you can get rid of the I prefix for the interfaces.
We're not using them anymore.